CRM/Event/Form/ManageEvent/EventInfo.php

```diff
@@ -33,18 +33,8 @@ public function preProcess() {
     $this->setSelectedChild('settings');
 
     $entityID = $this->_id ?: $this->_templateId;
-    if ($entityID) {
-      $this->assign('entityID', $entityID);
-      $eventType = CRM_Core_DAO::getFieldValue('CRM_Event_DAO_Event',
-        $entityID,
-        'event_type_id'
-      );
-    }
-    else {
-      $eventType = 'null';
-    }
+    $this->assign('entityID', $entityID);
```

Comment:
This still gives an error because in the template it's entityId. Hmm...

Reply:
Having said that, it's mergeable since it's better and not worse.

Reply:
@demeritcowboy sigh - so should I just remove that line? is it not used If so I will remove because it would be not worse & there would be less useless code

Reply:
It's used but as entityId. I didn't check how far back it goes but it's possible it has never worked, so yes there's an argument to remove it AND the .hlp/.tpl code.